syntax/forth.vim

```diff
@@ -137,13 +137,12 @@ syn match forthDebug "\<\~\~\>"
 syn keyword forthAssembler ASSEMBLER CODE END-CODE ;CODE FLUSH-ICACHE C,
 
 " basic character operations
-syn keyword forthCharOps (.) CHAR EXPECT FIND WORD TYPE -TRAILING EMIT KEY
+syn keyword forthCharOps (.) EXPECT FIND WORD TYPE -TRAILING EMIT KEY
 syn keyword forthCharOps KEY? TIB CR BL COUNT SPACE SPACES
 " recognize 'char (' or '[char] (' correctly, so it doesn't
 " highlight everything after the paren as a comment till a closing ')'
 syn match forthCharOps '\<char\s\S\s'
 syn match forthCharOps '\<\[char\]\s\S\s'
-syn region forthCharOps start=+."\s+ skip=+\\"+ end=+"+
```

Comment:
I assume that this is intended to cover strings: `." FOO"` Moved to the strings section.

Reply:
Actually, I am ready to back-track on this one. I think it is reasonable to assume, by convention, that any word that ends with `"` starts a string definition.

So a simplified definition could be:

`syn region forthString matchgroup=forthString start=+\S\+"\s+ end=+"+ end=+$+ contains=@Spell`

This would then cover `."`, `s"`, `c"` and anything that matches that pattern.

Reply:
I don't feel strongly about it either way.  A quick search of GitHub suggests exceptions exist but are rare.

If you want to reinstate it I'd just add a comment mentioning the standard words it catches and its greater purpose. 

In the worst case it will just erroneously mark the rest of the line as a string rather than the rest of the file as it previously did.